internal/server/handler_v2.go

```diff
@@ -242,5 +243,8 @@ func (s *Server) V2Observation(
 	localResp, remoteResp := <-localRespChan, <-remoteRespChan
 	// The order of argument matters, localResp is prefered and will be put first
 	// in the merged result.
-	return merger.MergeObservation(localResp, remoteResp), nil
+	mergedResp := merger.MergeObservation(localResp, remoteResp)
+	calculatedResps := v2observation.CalculateObservationResponses(ctx, s.store, mergedResp, s.cachedata.Load())
+	// mergedResp is preferred over any calculated response.
+	return merger.MergeMultipleObservations(append([]*pbv2.ObservationResponse{mergedResp}, calculatedResps...)...), nil
```

Comment:
Is it possible to write as `merger.MergeMultipleObservations(mergedResp, calculatedResps...)`?

Reply:
It says that there are too many arguments - is there another way to write this? For now I just separated out the inner append to make it more readable